sql/updates/latest-dev.sql

```diff
@@ -182,23 +182,51 @@ WHERE job_type = 'drop_chunks'
   AND job.id = c.job_id;
 
 -- migrate cagg jobs
+--- timescale functions cannot be invoked in latest-dev.sql
+--- this is a mapping for get_time_type
+CREATE FUNCTION ts_tmp_get_time_type(htid integer)
+RETURNS OID LANGUAGE SQL AS
+$BODY$
+  SELECT dim.column_type
+  FROM _timescaledb_catalog.dimension dim
+  WHERE dim.hypertable_id = htid and dim.num_slices is null
+        and dim.interval_length is not null;
+$BODY$;
+--- this is a mapping for _timescaledb_internal.to_interval
+CREATE FUNCTION ts_tmp_get_interval( intval bigint)
+RETURNS INTERVAL LANGUAGE SQL AS
+$BODY$
+   SELECT format('%sd %ss', intval/86400000000, (intval%86400000000)/1E6)::interval; 
+$BODY$;
+ 
 UPDATE
   _timescaledb_config.bgw_job job
 SET
-  application_name = format('%s [%s]', 'Continuous Aggregate Policy', id),
+  application_name = format('%s [%s]', 'Refresh Policy', id),
```

Comment:
thats inconsistent with the function name

Reply:
What is the convention for the application name?

Reply:
to be consistent with other names it should be `Refresh Continuous Aggregate Policy`

Reply:
modified.